R/checkMeasures.R

```diff
@@ -3,6 +3,13 @@ checkMeasures = function(measures, obj, aggr = NULL) {
     measures = list(getDefaultMeasure(obj))
   } else {
     measures = ensureVector(measures, n = 1L, cl = "Measure")
+    if (any(names(measures) == "na.rm")) {
```

Comment:
Because `na.rm` is passed within the measures argument in `resample()`. Hence it will be a subobject within the `measures` object. This call finds it, modifies it as needed (moves it to the according `prop` section of each measure) so that follow up functions will work as before. 

`measures = list(auc, mmce, na.rm = T))`